public/js/projectsConstructor.js

```diff
@@ -1,49 +1,48 @@
 'use strict';
 
-var projects = [];
+var app = app || {};
 
-function Project (projectsListObj) {
-  this.title = projectsListObj.title;
-  this.url = projectsListObj.url;
-  this.githubURL = projectsListObj.githubURL;
-  this.img = projectsListObj.img;
-  this.skillsUsed = projectsListObj.skillsUsed;
-}
+var projects = [];
```

Comment:
You should attach `projects` to `app` as well. Possibly as a property of `Project`.

Reply:
But its still an array holding all the projects right ? 

Does: 
``` app.Project(projects) = [];
 ``` look right ?

Reply:
You just attach `projects` as a property. So you could do `Project.projects = [];`, since `Project` gets attached to `app`, or `app.projects = projects;` at the bottom of the file if you do not want the array of projects to be a property on `Project`.